draft-ietf-quic-recovery.md

```diff
@@ -767,20 +767,22 @@ While in slow start, QUIC increases the congestion window by the
 number of bytes acknowledged when each acknowledgment is processed, resulting
 in exponential growth of the congestion window.
 
-QUIC exits slow start upon loss or upon increase in the ECN-CE counter.
-When slow start is exited, the congestion window halves and the slow start
-threshold is set to the new congestion window.  QUIC re-enters slow start
-any time the congestion window is less than the slow start threshold,
-which only occurs after persistent congestion is declared.
+QUIC MUST exit slow start and enter congestion avoidance upon loss or upon
+increase in the ECN-CE counter. When slow start is exited, the congestion
+window halves and the slow start threshold is set to the new congestion
+window.  QUIC re-enters slow start any time the congestion window is less
+than the slow start threshold, which only occurs after persistent congestion
+is declared.
 
 ## Congestion Avoidance
 
-Slow start exits to congestion avoidance.  Congestion avoidance uses an
-Additive Increase Multiplicative Decrease (AIMD) approach that increases
-the congestion window by one maximum packet size per congestion window
-acknowledged.  When a loss or ECN-CE marking is detected, NewReno halves
-the congestion window, sets the slow start threshold to the new
-congestion window, and then enters the recovery period.
+Congestion avoidance uses an Additive Increase Multiplicative Decrease (AIMD)
+approach that typically increases the congestion window by one maximum datagram
+size per congestion window acknowledged, and MUST NOT increase the congestion
```

Comment:
This MUST NOT kind of already says that QUICv1 cannot implement Cubic. Is that what we want?

Reply:
It does not, since this document allows an endpoint to implement a different controller than the one specified here. See [Section 7](https://quicwg.org/base-drafts/draft-ietf-quic-recovery.html#section-7-3):
```
If an endpoint uses a different controller than that specified in this document, the chosen controller MUST conform to the congestion control guidelines specified in Section 3.1 of [RFC8085].
```